.github/workflows/ci.yml

```
@@ -1,141 +1,141 @@
+          #   # https://github.com/orgs/community/discussions/19197#discussioncomment-10550689
+          #   # The conditional setup of QEMU may need to be adapted to not run on native arm,
+          #   # which will speed up the build process for arm64/v8 since no emulation will be needed.
+          #   RUNNER_TYPE: ubuntu-latest
+      fail-fast: false
+    name: ${{ matrix.name }}
+    timeout-minutes: 45
+    runs-on: ${{ matrix.RUNNER_TYPE }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

actionlint (1.7.7)

the runner of "docker/setup-qemu-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)